hkm/static/hkm/js/hkm-v2.js

```diff
@@ -339,6 +339,33 @@ palikka
   }
 
 })
+.define('app.feedback', ['jQuery', 'docReady'], function() {
+  $('#feedback-form').on('submit', function(event) {
+    event.preventDefault();
+
+    $.post('/record/feedback/', {
+      action: 'feedback',
+      csrfmiddlewaretoken: $('input[name="csrfmiddlewaretoken"]').val(),
+      content: $('#id_feedback-form-content').val(),
+      full_name: $('#id_feedback-form-full_name').val(),
+      email: $('#id_feedback-form-email').val(),
+      hkm_id: $('input[name="hkm_id"]').val(),
+    }).done(function(response) {
```

Comment:
I think `done` is only triggered if there is no error. Should there also be a `fail` option here? Or maybe replace `done` with `always`?

Reply:
My jQuery is a little rusty. After looking into it, I think you are right. I added `.fail` to catch errors.

Reply:
Yup, looking good!